server/utils/__tests__/ssr.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { shallow } from 'enzyme';
+import dataInitial from 'utils/storybook/data/dataInitial';
+import { pubData, attributionsData } from 'utils/storybook/data';
+import { generateMetaComponents } from '../ssr';
+
+Enzyme.configure({ adapter: new Adapter() });
+const defaultProps = {
+	initialData: dataInitial,
+	attributions: pubData.attributions,
+	title: 'Test Pub',
+};
+
+const MetaComponents = (props) => <>{generateMetaComponents(props)}</>;
```

Comment:
maybe use a react fragment, but im being picky

Reply:
`<></>` gets converted to a React.Fragment tag actually (which is why this line is needed). are you saying I should use the longhand `<React.Fragment>` version for readability?

Reply:
yeah but seriously dont if you dont want to

Reply:
nope just making sure i understood! i think the shorthand is nice where there's lots of html but that's definitely not the case here